schematron/v2.2.0/exemplary_files/L200_Audit.xml

```diff
@@ -258,7 +258,96 @@
         <auc:HVACSystems>
           <!-- 6.1.1.1.g/5.3.4.g (for every space function) -->
           <auc:HVACSystem ID="HVAC-A">
-            <auc:PrincipalHVACSystemType>Packaged Terminal Air Conditioner</auc:PrincipalHVACSystemType>
+            <auc:Plants>
+              <auc:CoolingPlants>
+                <auc:CoolingPlant ID="CoolingPlant-A">
+                  <auc:DistrictChilledWater>
+                    <auc:AnnualCoolingEfficiencyValue>123</auc:AnnualCoolingEfficiencyValue>
+                    <auc:AnnualCoolingEfficiencyUnits>COP</auc:AnnualCoolingEfficiencyUnits>
+                    <auc:Capacity>123</auc:Capacity>
+                    <auc:CapacityUnits>gpm</auc:CapacityUnits>
+                  </auc:DistrictChilledWater>
+                  <auc:CoolingPlantCondition>Good</auc:CoolingPlantCondition>
+                  <auc:YearInstalled>2020</auc:YearInstalled>
+                  <auc:PrimaryFuel>Other</auc:PrimaryFuel>
+                  <auc:BuildingAutomationSystem>true</auc:BuildingAutomationSystem>
+                  <auc:ControlSystemTypes>
+                    <auc:ControlSystemType>
+                      <auc:Digital></auc:Digital>
+                    </auc:ControlSystemType>
+                  </auc:ControlSystemTypes>
+                </auc:CoolingPlant>
+              </auc:CoolingPlants>
+            </auc:Plants>
+            <auc:HeatingAndCoolingSystems>
+              <auc:ZoningSystemType>Single zone</auc:ZoningSystemType>
+              <auc:HeatingSources>
+                <auc:HeatingSource ID="HeatingSource-A">
```

Comment:
Hey ted - idk how this slipped the radar until now, but I would say we also need to add checks for the following.

An `auc:HeatingSource` should have the following:
- `count(auc:HeatingSourceType/*) = 1`
- For `auc:HeatingSourceType/auc:Furnace`:
  - `auc:FurnaceType`
- For `auc:HeatingSourceType/auc:HeatPump`:
  - `auc:HeatPumpType`
  - `auc:HeatPumpBackupSystemFuel`

An `auc:CoolingSource` should have the following:
- `count(auc:CoolingSourceType/*) = 1`
- For `auc:CoolingSourceType/auc:DX`:
  - `auc:DXSystemType`
  - `auc:CompressorType`
  - `auc:CompressorStaging`
- For `auc:EvaporativeCooler`:
  - `auc:EvaporativeCoolingType`

Reply:
@corymosiman12 are this for _all_ heating/cooling sources, or only for those that don't reference a heating/cooling plant?

E.g. we currently have special requirements for `auc:HeatingSources/auc:HeatingSource[not(auc:HeatingSourceType/auc:HeatingPlantID)]` (same for cooling)

Reply:
I would narrate the checks as follows for a `auc:HeatingSource` (mirror for `auc:CoolingSource`):
- There must be an `auc:HeatingSourceType` defined
- For a heating source which is an `auc:Furnance`, it must define the `auc:FurnaceType`
... etc.

Reply:
My goof, Heating/CoolingPlantID are choices for Heating/CoolingSourceType. I'll add these tests

Reply:
Addressed here: 94e2a78

Reply:
Added to 211 spreadsheet as well under 6.2.1.3.a.1 0: Identify the System type

Reply:
Hey @macintoshpie, just added a few new things here:

![image](https://user-images.githubusercontent.com/28991233/96486837-ec65ce80-120a-11eb-842f-3253a66ce916.png)

Reply:
Addressed here a99187d465bfba0b0d876e6178385ba60a7f10dc